src/http/routing.rs

```diff
@@ -80,412 +145,534 @@ macro_rules! routes {
                 }))
             }
 
+            /// This returns the common identifiers, which are used in rate limit buckets.
+            /// If you are interested in less common identifiers,
+            /// please match the variants you are interested in.
+            #[must_use]
+            #[allow(unused_variables)] // prevent compiler from complaining about unused variables
```

Comment:
This comment is completely superfluous, it's kinda obvious.

Reply:
Removed